apps/test-app/frontend/package.json

```diff
@@ -35,10 +35,11 @@
     "@itwin/presentation-frontend": "^4.0.0-dev.85",
     "@itwin/webgl-compatibility": "^4.0.0-dev.85",
     "@opentelemetry/api": "^1.4.1",
+    "@types/copy-webpack-plugin": "^8.0.0",
     "@types/node": "^16.0.0",
     "@types/react": "^17.0.37",
     "@types/react-dom": "^17.0.0",
-    "copy-webpack-plugin": "^11.0.0",
+    "copy-webpack-plugin": "^9.0.0",
```

Comment:
Why did you need to downgrade `copy-webpack-plugin`?

Reply:
Some sub-dependency problems... 

`copy-webpack-plugin` [since version 10](https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/CHANGELOG.md#-breaking-changes-1) started using `globby@12`, which [delivers only ESM modules](https://github.com/sindresorhus/globby/releases/tag/v12.0.0). All our other dependencies use `glooby@11`. The two were colliding and causing build failures.